Source/missiles.cpp

```diff
@@ -1689,7 +1689,8 @@ void AddRing(Missile &missile, Point /*dst*/, Direction /*midir*/)
 
 void AddSearch(Missile &missile, Point /*dst*/, Direction /*midir*/)
 {
-	AutoMapShowItems = true;
+	if (missile._misource == MyPlayerId)
+		AutoMapShowItems = true;

```

Comment:
Consider leaving a blank line after this for readability.

Reply:
Good point to late